pom.xml

```diff
@@ -67,6 +67,11 @@
       <artifactId>authentication-tokens</artifactId>
       <version>1.3</version>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins.temp.jelly</groupId>
+      <artifactId>multiline-secrets-ui</artifactId>
```

Comment:
This is in 2.171 so why not simply adjust `jenkins.version` and skip this library?

Reply:
Can do. This PR predates the PR that was merged into 2.171.

Reply:
Right, that was just released I guess.